ansible/tasks/logging/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: windows
+  gather_facts: no
+  tasks:
+    - name: Ping test
+      win_ping:
+- hosts: windows
+  vars:
+    ruby_version: "RubyInstaller-2.4.7-1/rubyinstaller-devkit-2.4.7-1-x86.exe"
+    ruby_installer_path: "https://github.com/oneclick/rubyinstaller2/releases/download"
+    ruby_fetch_url: "{{ ruby_installer_path }}/{{ ruby_version }}"
+  tasks:
+   - name: Logging | Install Fluentd
+     include_tasks: install_fluentd.yaml
+   - name: Logging | Install Fluentd plugins
+     include_tasks: install_plugins.yml
```

Comment:
I don't see this file. I assume this is being added in #39. Won't it cause an error if we try running it?

Reply:
True. Will remove it